### PR TITLE
Updates from rspec-dev (2016-07-14)

### PR DESCRIPTION
**Pull request by JonRowe**
These are some updates, generated from rspec-dev's rake tasks.

### CODE REVIEW

**Comment by JonRowe** [comment]
The same broken backtrace filtering that had to be pended in the specs needed to be pended in the cukes, ok to merge @myronmarston ?

File: script/functions.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ function run_specs_and_record_done {
 
   # rspec-core needs to run with a special script that loads simplecov first,
   # so that it can instrument rspec-core's code before rspec-core has been loaded.
-  if [ -f script/rspec_with_simplecov ]; then
+  if [ -f script/rspec_with_simplecov ] && is_mri; then
```
Comment (myronmarston) [inline script/functions.sh]: Does simplecov not work on JRuby anymore?

**Comment by myronmarston** [comment]
Hmm, there's kinda a lot that you had to pend here.  I don't want these things to be marked as pending and just be left that way.  Do you have a plan for tackling them?

File: script/functions.sh (hunk 1 above)
Comment (JonRowe) [inline script/functions.sh]: I'm not sure we've ever tried to run it before, our other JRuby modes report themselves as `< 2` so miss this anyway, but `rspec-core` is reporting 0% coverage on JRuby 9.1.50

**Comment by JonRowe** [comment]
Yeah I wasn't overly happy doing it but I feel as this is a build steps PR and I didn't have the time to fix them straight away, would you be happy if I opened an issue to keep track of them?

**Comment by myronmarston** [comment]
> Yeah I wasn't overly happy doing it but I feel as this is a build steps PR and I didn't have the time to fix them straight away, would you be happy if I opened an issue to keep track of them?

Sure.  It'd be great if the issues provided some detail about what the failure is.

**Comment by JonRowe** [comment]
Ok to merge otherwise?

**Comment by myronmarston** [comment]
> Ok to merge otherwise?

Yes

**Pull request merged**